main.tf

```diff
@@ -1 +1,346 @@
+###
+### LOCALS
+###
 
+locals {
+  name_id         = "${var.name}-${random_string.this.result}"
+  bucket          = "vault-salt-formula"
+  ami_name_filter = "spel-minimal-centos-7-hvm-*.x86_64-gp2"
+  ami_name_regex  = "spel-minimal-centos-7-hvm-\\d{4}\\.\\d{2}\\.\\d{1}\\.x86_64-gp2"
+  vpc_id          = "${data.aws_subnet.lb.0.vpc_id}"
+  policy_name     = "INSTANCE_VAULT_${data.aws_caller_identity.current.account_id}"
+  ssm_root_path   = "vault/${var.environment}/${data.aws_caller_identity.current.account_id}/${var.name}"
+  plain_domain    = "${replace(element(split(":", var.vault_dns_address), 1), "////", "")}"
+  public_ip       = "${chomp(data.http.ip.body)}/32"
+  allow_inbound   = "${compact(distinct(concat(list(local.public_ip), var.additional_ips_allow_inbound)))}"
+
+  tags = {
+    Environment = "${var.environment}"
```

Comment:
users often have their own mandatory tags, so be sure to establish a `tags` variable and merge that wherever you apply tags